evaluation.py

```diff
@@ -237,7 +237,97 @@ def predict_keypoints(self,generator, location):
+                            dist_list.append(dist)
+
+                    correct_keypoints["nose"] += dist_list[0]<threshold
+                    correct_keypoints["left_eye"] += dist_list[1]<threshold
+                    correct_keypoints["right_eye"] += dist_list[2]<threshold
+                    correct_keypoints["left_ear"] += dist_list[3]<threshold
+                    correct_keypoints["right_ear"] += dist_list[4]<threshold
+                    correct_keypoints["left_shoulder"] += dist_list[5]<threshold
+                    correct_keypoints["right_shoulder"] += dist_list[6]<threshold
+                    correct_keypoints["left_elbow"] += dist_list[7]<threshold
+                    correct_keypoints["right_elbow"] += dist_list[8]<threshold
+                    correct_keypoints["left_wrist"] += dist_list[9]<threshold
+                    correct_keypoints["right_wrist"] += dist_list[10]<threshold
+                    correct_keypoints["left_hip"] += dist_list[11]<threshold
+                    correct_keypoints["right_hip"] += dist_list[12]<threshold
+                    correct_keypoints["left_knee"] += dist_list[13]<threshold
+                    correct_keypoints["right_knee"] += dist_list[14]<threshold
+                    correct_keypoints["left_ankle"] += dist_list[15]<threshold
+                    correct_keypoints["right_ankle"] += dist_list[16]<threshold
+                    dist_list = []
+
+        samples = len(list_of_predictions)
+        print("Percentage of Correct Key Points (PCK)\n")
+        print("Nose: {:.2f}".format(correct_keypoints["nose"]/samples))
+        print("Left Eye: {:.2f}".format(correct_keypoints["left_eye"]/samples))
+        print("Right Eye: {:.2f}".format(correct_keypoints["right_eye"]/samples))
+        print("Left Ear: {:.2f}".format(correct_keypoints["left_ear"]/samples))
+        print("Right Ear: {:.2f}".format(correct_keypoints["right_ear"]/samples))
+        print("Left Shoulder: {:.2f}".format(correct_keypoints["left_shoulder"]/samples))
+        print("Right Shoulder: {:.2f}".format(correct_keypoints["right_shoulder"]/samples))
+        print("Left Elbow: {:.2f}".format(correct_keypoints["left_elbow"]/samples))
+        print("Right Elbow: {:.2f}".format(correct_keypoints["right_elbow"]/samples))
+        print("Left Wrist: {:.2f}".format(correct_keypoints["left_wrist"]/samples))
+        print("Right Wrist: {:.2f}".format(correct_keypoints["right_wrist"]/samples))
+        print("Left Hip: {:.2f}".format(correct_keypoints["left_hip"]/samples))
+        print("Right Hip: {:.2f}".format(correct_keypoints["right_hip"]/samples))
+        print("Left Knee: {:.2f}".format(correct_keypoints["left_knee"]/samples))
+        print("Right Knee: {:.2f}".format(correct_keypoints["right_knee"]/samples))
+        print("Left Ankle: {:.2f}".format(correct_keypoints["left_ankle"]/samples))
+        print("Right Ankle: {:.2f}".format(correct_keypoints["right_ankle"]/samples))
```

Comment:
small nit: using a tab or spaces might make this row easier to read once it prints out